src/GeositeFramework/plugins/launchpad/templates.html

```diff
@@ -23,7 +28,12 @@
                 <li>
                     <div>
                         <h4><%= scenario.title %></h4>
-                        <p><%= scenario.description %></p>
+                        <div class="truncate-container">
+                            <p><%= scenario.description %></p>
+                            <% if (scenario.description.length > 140) { %>
+                                <button class="button truncate-toggle i18n" data-i18n="[title]See Less" title="See Less"></button>
```

Comment:
Not sure if this was intentional, but the button markup is a little different for scenarios compared to plugins. I noticed that the tooltip doesn't always match the button text.

![image](https://cloud.githubusercontent.com/assets/43062/21297159/9ae8e7b2-c548-11e6-8f27-cced0960dad4.png)